### ISSUE TITLE
DESCRIPTIONS says to use lazy data but it is not set up properly

### ISSUE THREAD

**Issue by Kdreval** [morinlab/GAMBLR#174]
```
❯ checking LazyData ... WARNING
    LazyData DB of 8.3 MB without LazyDataCompression set
    See §1.1.6 of 'Writing R Extensions'
```

**Comment by Kdreval** [comment]
This seem to need a fix of adding
```
LazyDataCompression: xz
```
To the `DESCRIPTION` file

**Comment by Kdreval** [comment]
Closing as fixed in #177
